guides/v2.1/cloud/env/environment-vars_magento.md

```diff
@@ -8,16 +8,28 @@ functional_areas:
-| ADMIN_LOCALE    | Default—`en_US` Specifies the default locale used by the Magento Admin.                                                                                                                                                                                                                                |
-| ADMIN_PASSWORD  | Password for the administrative user. When the project is created a random password is generated and an email is sent to the Project Owner. During project creation, the Project Owner should have already changed the password. You might need to contact the Project Owner for the updated password. |
-| ADMIN_URL       | The relative URL to access the Admin panel. For example: `/admin`. For security reasons, we recommend you choose a value other than `admin`,`backend`, or another term that is easy to guess.                                                                                                          |
-| ADMIN_USERNAME  | Username for the administrative user. The administrative user can create other users, including other administrative users. The default hardcoded username is the Project Owner email address. You can use this value, or change it to another secure username.                                        |
+| `ADMIN_USERNAME`|A username for the administrative user with the ability to create other users, including administrative users. During initial project provisioning, the username defaults to the {{ site.data.var.ece }} License Owner email address. You can use the default value, or change it to a secure username.|
+|`ADMIN_EMAIL`|Email address for the administrative user. This address is used to send password reset notifications.|
+|`ADMIN_PASSWORD`|Password for the administrative user. When the project is created a random password is generated and an email is sent to the {{ site.data.var.ece }} License Owner. During project creation, the License Owner should have already changed the password. You might need to contact the License Owner for the updated password.|
+| `ADMIN_LOCALE`    | Default—`en_US` Specifies the default locale used by the Magento Admin.
 {:style="table-layout:auto;"}
+
+
+**Admin URL**
+
+Use the following environment variable to secure access to your Magento Admin UI. If specified, this value overrides the default URL during installation.
+
+**`ADMIN_URL`**—The relative URL to access the Magento Admin UI. For example: `/admin`. For security reasons, we recommend that you choose a value other than `admin`,`backend`, or another term that is easy to guess.
+
+If you need to change the Admin URL after installation, connect to your environment using ssh and use the Magento CLI [`magento setup:config`]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands-deployment.html) command to change the URL. 
```

Comment:
I just noticed this command is highlighted differently than the previous one for admin:user.